app/views/organisations/_promotional_features.html.erb

```diff
@@ -1,7 +1,7 @@
 <% if @documents.has_promotional_features? %>
-<section class="organisation__margin-bottom organisation__promotions">
+<section class="organisation__margin-bottom organisation__section-wrap">
   <% @documents.promotional_features.each do |feature| %>
-    <div class="<%= feature[:parent_column_class] %>">
+    <div class="<%= feature[:parent_column_class]%>">
```

Comment:
Is it possible to remove this wrapping div for single item promos? It's now redundant in that context. 

There are also similar redundant <divs> at lines 13(19) and 15(17) which aren't part of this PR but could also be removed for clarity of output.

However, I appreciate that this would incur extra logic for the sake of slightly cleaner HTML and may not be worthwhile. They're not doing any harm by being left in the output HTML.

Reply:
Thanks Matt, I'll make that change

Reply:
Removing the outer div at line 4 breaks some styling - the black dividing line is truncated. So I've left that as is. But have removed the inner divs when only one item is present see https://github.com/alphagov/collections/pull/3096/commits/4818e9fa70b36b58f1121fc0470ac6a6683fd84d